server/src/domain/game.rs

```diff
@@ -4,10 +4,18 @@ use super::player::Player;
 // Types //
 ////////////////////////////////////////////////////////////////
 
-pub struct Game {
+pub enum Game {
+    Lobby(LobbyModel),
+}
+
+pub struct LobbyModel {
+    metadata: Metadata,
+    host: Player,
+    opponents: Vec<Player>,
+}
```

Comment:
Before there was `players: Vec<Player>`, but, it shouldnt be logically possible for there to be zero players. The lobby starts off with at least one player, that being the host.